### PR TITLE
fix(DB/Creature): Port Underbog Colossus immunities from Mangos

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/6240

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

mangos wotlk db

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by elthehablo — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1699978192145266700.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Underbog Colossus
+UPDATE `creature_template` SET `unit_flags` = `unit_flags`|32832, `mechanic_immune_mask` = `mechanic_immune_mask`|42090322, `detection_range` = 30 WHERE (`entry` = 21251);
```
[the remark this answers is not in the document]
Comment (elthehablo) [reply data/sql/updates/pending_db_world/rev_1699978192145266700.sql:2]: > If you want to check the immunities you can always see it in keira. There is 0 chance I'll be doing them 1 by 1.

I literally gave you a button to press to fix it :(

[the remark this answers is not in the document]
Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1699978192145266700.sql:2]: Breaking them down is the whole point of using the bitwise operator as oposed to just a regular update query

**Comment by elthehablo** [comment]
pls update so we can merge this :) useful for PTR testing

**Comment by elthehablo** [comment]
Is that a threat? 😤

**Review by elthehablo — APPROVED** [review]

**Pull request merged**